src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1808,84 +1798,91 @@ def get_state(self):
     getState = __call__ = get_state
 
 
-class PyQuery:
-    """Query class using the StateEngine-based DatamodelService as the backend. Use this
-    class instead of directly calling the DatamodelService's method.
+class _PyAction:
```

Comment:
Since this is a base class of other public classes, it is also public class. The leading underscore should be omitted.